aiobotocore/client.py

```diff
@@ -183,6 +186,17 @@ def get_paginator(self, operation_name):
                 self._cache['page_config'][actual_operation_name])
             return paginator
 
+    if PY_35:
+        @asyncio.coroutine
+        def __aenter__(self):
```

Comment:
Look good! we need some tests for that in separate file, because this is only python3.5 ((